ui/src/app/components/resource-list.tsx

```diff
@@ -265,7 +263,7 @@ export const ResourceList = fp.flow(withCdrVersions())((props: Props) => {
               field='formattedLastModified'
               header='Last Modified Date'
               style={styles.modifiedDateColumn}
-              sortField={'lastModifiedDateAsString'}
```

Comment:
We incorrectly assumed this was a year-first string like '2024-02-02' but it was not!

Since the format is actually 02/02/24, it sorted Month -> Day -> Year